contracts/TokenBalanceOracle.sol

```diff
@@ -57,13 +57,7 @@ contract TokenBalanceOracle is AragonApp, IACLOracle {
     *     with 'authP(SOME_ACL_ROLE, arr(sender))', typically set to 'msg.sender'.
     */
     function canPerform(address, address, bytes32, uint256[] _how) external view returns (bool) {
-        require(_how.length > 0, ERROR_SENDER_MISSING);
-        require(_how[0] < 2**160, ERROR_SENDER_TOO_BIG);
-        require(_how[0] != 0, ERROR_SENDER_ZERO);
-
-        address sender = address(_how[0]);
-
-        uint256 senderBalance = token.balanceOf(sender);
-        return senderBalance >= minBalance;
+        require(_how.length > 0 && _how[0] < 2**160 && _how[0] != 0, ERROR_CANT_PERFORM);
```

Comment:
I was thinking if we could remove the `_how[0] < 2**160`  check? Since we are casting it to an address in the line above. Not sure what potential problem could arise from passing a bigger hex number though.